input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,395 @@
+            <td style="width: 33.33%;"><a href="https://hl7.org/fhir/uv/ipa/STU1/StructureDefinition-ipa-medicationrequest.html">IPA-MedicationRequest</a></td>
+            <td style="width: 33.33%;">MedicationRequest.requester type differs: IPA prohibits reference to Organization and RelatedPerson resources.</td>
         </tr>
+    </tbody>
+</table>
+
+#### International Patient Summary 1.1.0
+
+This table provides detailed explanations for instances where AU Core profiles do not comply with the corresponding profiles from International Patient Summary.
+
+<table border="1" style="width: 100%; margin: auto; border-collapse: collapse;">
+    <thead>
         <tr>
-            <td width="93" valign="top">
-                <p>
-                    <a
-                        href="StructureDefinition-au-core-medicationrequest.html"
-                    >
-                        AU Core MedicationRequest
-                    </a>
-                </p>
-            </td>
-            <td width="81" valign="top">
-                <p>
-                    <a
-                        href="http://hl7.org/fhir/uv/ipa/StructureDefinition/ipa-medicationrequest"
-                    >
-                        IPA-MedicationRequest
-                    </a>
-                </p>
-            </td>
-            <td width="111" valign="top">
-                <p>
-                    <img src="https://hl7.org/fhir/R4/assets/images/tick.png"/> if requester is supplied as a reference to Practitioner, PractitionerRole or Patient resource
-                </p>
-            </td>
-            <td width="68" valign="top">
-                <p>
-                    <a
-                        href="http://hl7.org/fhir/uv/ips/StructureDefinition/MedicationRequest-uv-ips"
-                    >
-                        Medication Request (IPS)
-                    </a>
-                </p>
-            </td>
-            <td width="130" valign="top">
-                <p>
-                    <img src="https://hl7.org/fhir/R4/assets/images/tick.png"/> if patient is supported as
-                    MedicationRequest.subject.reference
-                </p>
-            </td>
-            <td width="78" valign="top">
-                <p>
-                    <a
-                        href="http://hl7.org/fhir/us/core/StructureDefinition/us-core-medicationrequest"
-                    >
-                        US Core MedicationRequest
-                    </a>
-                </p>
-            </td>
-            <td width="137" valign="top">
-                <p>
-                    <img src="https://hl7.org/fhir/R4/assets/images/tick.png"/> if core FHIR MedicationRequest Category codes are
-                    supplied in MedicationRequest.category
-                </p>
-            </td>
+            <th style="width: 33.33%;">AU Core</th>
+            <th style="width: 33.33%;">International Patient Summary 1.1.0</th>
```

Comment:
1. Instead of Comment it should be "Additional Requirements"
2. Add a column for element
3. Apply across the tables.
4. Refactor what was Comment section a bit to be shorter / clearer.

Reply:
1. Done
2. Done
3. Done

Reply:
1. Confirmed
2. Confirmed
3. Confirmed
4. Confirmed